selectrum-prescient.el

```diff
@@ -117,6 +117,73 @@ For use on `selectrum-candidate-selected-hook'."
     (remove-hook 'selectrum-candidate-inserted-hook
                  #'selectrum-prescient--remember)))
 
+;;;; Commands
+(defvar selectrum-prescient-filter-toggle-map (make-sparse-keymap)
+  "A keymap of commands for toggling Prescient filters in Selectrum.
+The toggling of commands is temporary and does not affect the
+default filtering settings determined by `prescient-filter-method'.")
+;; Create a binding similar to the Isearch toggles.
+(defvar selectrum-minibuffer-map)
```

Comment:
These `defvar` / `declare-function` forms can go in `stub/selectrum.el` with the others, so they're all centralized.